{
    "basename": "asr-generics_error_02-d614928",
    "cmd": "lpython --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/errors/generics_error_02.py",
    "infile_hash": "0cd9c07bbe28138a3cca57b750b625c528773ac79c36b15e5dc5830a",
    "outfile": null,
    "outfile_hash": null,
    "stdout": null,
    "stdout_hash": null,
    "stderr": "asr-generics_error_02-d614928.stderr",
    "stderr_hash": "ed8e1b5bb3e56a2855182951d7e800079399fd76e100d4aa9749d611",
    "returncode": 2
}